types/wordpress__components/button/index.d.ts

```diff
@@ -35,7 +38,7 @@ declare namespace Button {
         /**
          * Renders a toggled button style.
          */
-        isToggled?: boolean;
+        isPressed?: boolean;
```

Comment:
Just pointing out that this will be a breaking change in that builds will break after updating this package and they were actually using `isToggled`. However you could also argue that this is just a bug fix, since it doesn't look like `isToggled` ever did anything.

Does it make sense to leave the `isToggled` type there but mark it as `@deprecated`? That way when folks update from `@types/wordpress__components@9.8.5` to `@types/wordpress__components@9.8.x` there builds won't break which is expected for a minor version bump. Then when we can remove `isToggled` in `@types/wordpress__components@10.0.0`.

Reply:
I'm not familiar with `@wordpress/components`. Was `isToggled` ever a valid option in this or prior versions of the package? If not, then its probably safe to remove, though I concur with `@p-jackson` it might be worth leaving in with an `@deprecated` pointing to the correct property name. Something like this:

```suggestion
        isPressed?: boolean;
        /** @deprecated Use `isPressed` instead. */
        isToggled?: never;
```
By using `never`, anyone who sets `{ isToggled: true }` would get an error that would lead them to look at the property and ideally see that it is marked as deprecated.